ai/bert/Cargo.toml

```diff
@@ -22,9 +22,7 @@ tract-onnx = "0.18.3"
 onnxruntime = { version = "0.0.13", optional = true }
 
 [dev-dependencies]
-criterion = { workspace = true }
```

Comment:
❓:wrench: why is this removed? without it the `bert` bench declared below fails to compile.